expo-zustand-styled-components/src/hooks/stores/useIssuesStore.ts

```diff
@@ -18,53 +18,20 @@ interface Issues {
 }
 
 interface IssueStore {
-  isLoading: boolean;
   error?: string;
-  before?: string | null;
-  after?: string | null;
-  issues: Issues;
-  setLoading: (value: boolean) => void;
-  setIssues: (value: Issues) => void;
-  setBefore: (value: string) => void;
-  setAfter: (value: string) => void;
-  setErrorMsg: (value: string) => void;
-  resetBeforeAndAfter: () => void;
+  issues?: Issues;
+  isLoading: boolean;
+  _issues: Map<string, Issues>;
```

Comment:
I am a bit confused about this. Why do we have this property and why is it a map and how is is different from `issues?: Issues`?

Reply:
temporal caching